### DISCUSSION TITLE
Opening Btop4Win without it opening a new window

### DISCUSSION THREAD

**Discussion by ghost** [aristocratos/btop4win#21] [Q&A]
I installed Btop4Win via Scoop, was wondering if there was a command line way to make it display in the Powershell  window I opened it in (like it does when I open it in a terminal on Linux or Mac) instead of it opening in a separate new window?   Thanks in advance for any constructive replies.

**Comment by aagha** [comment]
I have the same issue and would like to close this window:

![image](https://github.com/aristocratos/btop4win/assets/5522103/dafd0ee1-7c26-40b4-b6d3-c3dc9aea81bf)

**Comment by lane-ftw** [comment]
It seems to be possible, as shown in this screenshot on the main page. Would also like this to stay in my terminal window rather than making it's own.
EDIT: Run it in an admin prompt to keep it in the same window it was run from, as it needs admin rights to run. When running it from a standard shell, it spawns an admin window to run itself in.
![](https://github.com/aristocratos/btop4win/raw/master/Img/services.png)
